src/initializers/koa/add-request-id.ts

```diff
@@ -1,7 +1,7 @@
 import { Context, Middleware } from 'koa';
 
 export default function(config): Middleware {
-  return async function(ctx: Context, next: () => void) {
+  return async function addRequestId(ctx: Context, next: () => void) {
```

Comment:
Is this added for logging middleware name to newrelic?

Reply:
Yes, it would be nice if newrelic could log these